desktop/src/main/java/bisq/desktop/main/market/offerbook/OfferBookChartView.java

```diff
@@ -146,14 +147,20 @@ public OfferBookChartView(OfferBookChartViewModel model, Navigation navigation,
     public void initialize() {
         createListener();
 
+        // Header with currency combobox and checkbox to show extreme values
+        GridPane topGrid = new GridPane();
```

Comment:
Please also remove the additional GridPane again, so we don't use more containers as necessary.